README.md

```diff
@@ -1,124 +1,115 @@
-llvm: >= 15
-python: > 3.8 and  <= 3.11
-```
+## Why Milvus
 
-MacOS systems with Apple Silicon (Monterey 12.0.1 or later recommended):
-```bash
-go: >= 1.21 (Arch=ARM64)
-cmake: >= 3.26.4
-llvm: >= 15
-python: > 3.8 and  <= 3.11
-```
+Milvus is designed to handle vector search at scale. Users can store vectors, which are numerical representations of unstructured data, together with other scalar data types such as integers, strings, and JSON objects, to conduct efficient vector search with metadata filtering or hybrid search. Here are why users choose Milvus as vector database:
 
-Clone Milvus repo and build.
+**High Performance at Scale and High Availability**  
+  * Milvus features a [distributed architecture](https://milvus.io/docs/architecture_overview.md ) that separates [compute](https://milvus.io/docs/data_processing.md#Data-query) and [storage](https://milvus.io/docs/data_processing.md#Data-insertion). Milvus can horizontally scale and adapt to diverse traffic patterns, by independent scaling worker nodes for optimized performance in read-heavy or write-heavy workloads. The stateless microservices on K8s allow [quick recovery](https://milvus.io/docs/coordinator_ha.md#Coordinator-HA) from failure, ensuring high availability. [Replication](https://milvus.io/docs/replica.md) further enhances fault tolerance and throughput by loading data segments on multiple query nodes. Milvus also optimizes vector search at high metadata filtering rate. See [performance benchmark](https://zilliz.com/vector-database-benchmark-tool).
 
-```bash
-# Clone github repository.
-$ git clone https://github.com/milvus-io/milvus.git
 
-# Install third-party dependencies.
-$ cd milvus/
-$ ./scripts/install_deps.sh
+**Support for Various Vector Index Types and Hardware Acceleration**  
+  * Milvus supports vector index types that are optimized for different scenarios, including HNSW, IVF, FLAT (brute-force), and DiskANN, with [quantization-based](https://milvus.io/docs/index.md?tab=floating#IVFPQ) variations and [mmap](https://milvus.io/docs/mmap.md). Additionally, Milvus implements instruction-level acceleration to enhance vector search performance and supports GPU indexing, such as NVIDIA's [CAGRA](https://github.com/rapidsai/raft).
 
-# Compile Milvus.
-$ make
-```
+**Sparse Vector for Full Text Search and Hybrid Search**
+  * In addition to semantic search via dense vector embeddings, Milvus also supports full text search with [sparse vector](https://milvus.io/docs/sparse_vector.md). User can also combine dense vector and sparse vector through multi-vector feature and perform hybrid search. Milvus supports up to 10 vector fields in a single collection. Milvus supports hybrid search on dense and sparse vector columns, or multiple dense vector columns. The data from multi-path retrieval can be merge and rerank with Reciprocal Rank Fusion (RRF) and Weighted Scoring. For details, refer to [Hybrid Search](https://milvus.io/docs/multi-vector-search.md).
```

Comment:
This is too detailed and might change in the near future.
Just say that milvus support add User defined function to rerank data from multiple search would be good.